### DISCUSSION TITLE
Fantastic project !

### DISCUSSION THREAD

**Discussion by linuxguy123** [sle118/squeezelite-esp32#119] [General]
Just wanted to say it.  I've been looking for something like this for a long time.   I'll be making a ton of these to put wireless speakers around my house. 

Good documentation on the Github project homepage too.  Well done.

**Comment by ElFishi** [comment]
pics or it didn't happen

**Reply by ismal** [reply]
Indeed, this is a great project! Everything works both with different i2s DACs that I tired and SPI OLED and TFT screens!

Rotation of the screen 90 degrees also works, but I had to "check" the option "Rotate 180 degrees" and "Flip vertically". In any case, shouldn't be called 90 and not 180 degrees?

**Comment by sle118** [comment]
Thank you for the comments! I have to give all credits for documentation to Philippe, who was able to keep it current!
